### PR TITLE
Fix some choices not working for `requireenterconfirm`

### LINKED ISSUE #94: Doesn't let me edit PKGBUILD with requireenterconfirm = no
When the config `requireenterconfirm = no` is active and I press `y` when pikaur asks me whether I want to edit the PKGBUILD, it just doesn't care and continues ;-)

### PR DESCRIPTION
**Pull request by JakeStanger**
The list of answers could contain capitals, where input could not. This meant they were not equal when comparing them, so no answer was returned.

_edited:_
fixes #94

### CODE REVIEW

File: pikaur/prompt.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def read_answer_from_tty(question: str, answers: str = 'Yn') -> str:
         if ord(answer) == 13:
             answer = default
             return default
-        if answer in answers:
+        if answer in [a.lower() for a in answers]:
```
Comment (actionless) [inline pikaur/prompt.py]: `answers.lower()`?

Comment (JakeStanger) [reply pikaur/prompt.py]: 🤦forgot answers isn't a list despite looking straight at it.

**Comment by actionless** [comment]
also please rebase your change on top of master

(in the future creating a feature branch instead of working on the master of your repository copy will help to avoid such problems)

**Comment by actionless** [comment]
to fix that do 
```
git fetch --all
git checkout upstream/master
git checkout -b my-feature
git cherry-pick ccf3ed34e702f0b09e47dac9355cc524830205d0
git push origin HEAD
```

**Pull request closed without merging**